menas/src/main/resources/application.properties.template

```diff
@@ -73,10 +73,6 @@ menas.environment=Dev
 
 menas.spark.master=local[1]
 
-# The template for Lineage (Spline) display. If empty Lineage button is not shown.
-# Avoiding the http(s) protocol will cause that the same protocol will be used as for Menas itself, avoiding issues
-spline.urlTemplate=//localhost:8080/spline/dataset/lineage/_search?path=%s&application_id=%s
```

Comment:
Since you removed this. I cannot build the project as it is a mandatory property for Run Service